api/models.py

```diff
@@ -451,16 +451,22 @@ def perform_migration(self) -> None:
         existing_submissions = Submission.objects.filter(completed_by=self.old_user)
         self.affected_submissions.add(*existing_submissions)
 
+        # need to process transcriptions first because the submissions they're
+        # linked to are about to change
+        transcriptions = Transcription.objects.filter(
+            submission__in=existing_submissions, author=self.old_user
+        )
+        transcriptions.update(author=self.new_user)
```

Comment:
Because we keep track of the submissions we act on (for revert purposes) it makes more sense to only modify the transcriptions attached to the submissions we're tracking so that we don't overreach. I agree that _in most cases_ it'll be everything attached to the user, but I can't guarantee that.